src/util/pagination-utils.ts

```diff
@@ -1,29 +1,15 @@
-import * as isEqual from 'lodash.isequal';
```

Comment:
can you check if `isEqual` is used anywhere else in the lib? if not please remove the dependency